File: Easy/Java/2544. Alternating Digit Sum.java
class Solution {
    public int alternateDigitSum(int n) {
        String num = Integer.toString(n);
        int sum = 0;

        for(int i = 0; i < num.length(); i++){
            if(i % 2 == 0)
                sum += Character.getNumericValue(num.charAt(i));
            else
                sum -= Character.getNumericValue(num.charAt(i));
        }

        return sum;
    }
}

File: Easy/JavaScript/2544. Alternating Digit Sum.js
var alternateDigitSum = function(n) {
    let num = n.toString();
    let sum = 0;

    for(let i = 0; i < num.length; i++){
        if(i % 2 == 0)
            sum += parseInt(num.charAt(i));
        else
            sum -= parseInt(num.charAt(i));
    }

    return sum;
};
